### PR TITLE
Upgrade Netty version

### PR DESCRIPTION
**Pull request by varunnvs92**
Upgrade Netty dependencies
Replace https://github.com/aws/aws-sdk-java-v2/pull/1237

Some of the forked classes are deprecated. Removed them and made code changes as recommended in 
https://netty.io/news/2019/06/28/4-1-37-Final.html

### CODE REVIEW

File: http-clients/netty-nio-client/src/main/java/io/netty/handler/codec/http2/ForkedHttp2StreamChannelBootstrap.java [hunk 1]
```diff
@@ -44,6 +45,9 @@
     private final Channel channel;
     private volatile ChannelHandler handler;
 
+    // Cache the ChannelHandlerContext to speed up open(...) operations.
+    private volatile ChannelHandlerContext multiplexCtx;
+
     public ForkedHttp2StreamChannelBootstrap(Channel channel) {
```
Comment (dagnir) [inline http-clients/netty-nio-client/src/main/java/io/netty/handler/codec/http2/ForkedHttp2StreamChannelBootstrap.java]: Do you know what the changes we made in the class are compared to the one shipped with Netty? Would be cool if we could get rid of this one too!

Comment (varunnvs92) [reply http-clients/netty-nio-client/src/main/java/io/netty/handler/codec/http2/ForkedHttp2StreamChannelBootstrap.java]: Verified the diff between our forked file and 4.1.36 file. Both are same except the naming changes. Safe to delete this class too. will update pr.

**Comment by marcinczeczko** [comment]
@dagnir @varunnvs92 - any plans to release netty upgrade ?

**Comment by zoewangg** [comment]
@marcinczeczko The PR has been merged and the change will be included in the next release.

**Comment by marcinczeczko** [comment]
@zoewangg Thanks for the update

**Pull request merged**